public/static/js/settings.js

```diff
@@ -80,10 +89,42 @@ mciModule.controller('SettingsCtrl', ['$scope', '$http', '$window', 'notificatio
       });
   }
 
-  $scope.updateUserSettings = function(new_tz, use_spruce_options) {
+  $scope.onOptOutChange = function(){
```

Comment:
What I meant before is that you can set `ng-hide="use_spruce_options.patch_page"` on an element that contains all the feedback fields. That way you don't need to write any of this

Reply:
The reason I chose to do it differently is because when you just use ng-hide, the form fields are always shown when the checkbox is checked, even if it's checked initially. I only want those fields to be shown if the checkbox state goes from unchecked to checked.

Reply:
You can change the condition to check whether or not it was checked initially. My main point is that ng-hide should be able to do whatever you need it to with fewer lines of code